site/content/en/docs/admission-check-controllers/provisioning.md

```diff
@@ -12,6 +12,8 @@ The controller is part of Kueue. It is enabled by default. You can disable it by
 
 The Provisioning Admission Check Controller is supported on [Kubernetes cluster-autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) versions 1.29 and later. However, some cloud-providers may not have an implementation for it.
 
+Check the list of supported Provisioning Classes and prerequisite for them in [ClusterAutoscaler documentation](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md#supported-provisioningclasses).
```

Comment:
Can you also mention it in line 45? And also clarify that queue-provisioining.gke.io is a cloud-specific class. The current wording is unclear when it says "Check x for details". Where would I check this? :)

I'm also tempted to change line 39 to use an OSS class, but we can also leave it for now, as it's the only mature one at the moment.

Reply:
Updated, I replaced the class name in line 39, so there is no need to clarify that queue-provisioining.gke.io is a cloud-specific class. However now I'm thinking that we probably need a list of cloud-specific classes in CA documentation.

Reply:
Let's add here:
Also check your cloud provider's documentation for other ProvisioningRequest engines they support.

Reply:
Done